src/ray/raylet/lineage_cache.cc

```diff
@@ -152,16 +152,14 @@ const std::unordered_set<TaskID> &Lineage::GetChildren(const TaskID &task_id) co
   }
 }
 
-LineageCache::LineageCache(const ClientID &client_id,
-                           gcs::TableInterface<TaskID, TaskTableData> &task_storage,
-                           gcs::PubsubInterface<TaskID> &task_pubsub,
+LineageCache::LineageCache(std::shared_ptr<gcs::RedisGcsClient> gcs_client,
                            uint64_t max_lineage_size)
-    : client_id_(client_id), task_storage_(task_storage), task_pubsub_(task_pubsub) {}
+    : gcs_client_(gcs_client) {}
 
 /// A helper function to add some uncommitted lineage to the local cache.
 void LineageCache::AddUncommittedLineage(const TaskID &task_id,
                                          const Lineage &uncommitted_lineage) {
-  RAY_LOG(DEBUG) << "Adding uncommitted task " << task_id << " on " << client_id_;
+  RAY_LOG(DEBUG) << "Adding uncommitted task " << task_id;
```

Comment:
Keep printing the node ID. This is useful for debugging.

Reply:
done